app/stylesheets/SplitCost.scss

```diff
@@ -0,0 +1,12 @@
+.tilt-icon.tilt-icon-split-cost {
+  width: 23px;
+  height: 25px;
+  fill: white;
+}
+.outline {
+  stroke: #CFD6DC;
+  stroke-width: 2px;
+}
+.bottom-fold {
+  fill: #CFD6DC;
```

Comment:
since this color is used several places now, do you think it makes sense to put into _colors.scss?